pandas/tests/series/test_operators.py

```diff
@@ -960,8 +960,44 @@ def test_timedelta64_ops_nat(self):
         assert_series_equal(timedelta_series / nan,
                             nat_series_dtype_timedelta)
 
+    def test_operators_timedelta64_with_timedelta(self):
+        # smoke tests
+        td1 = Series([timedelta(minutes=5, seconds=3)] * 3)
```

Comment:
we certainly have *some* of these, can you remove the existing ones if you are consolidating.

these also seem oddly specific, would parametrize on Timedelta, TDI as well here

Reply:
> we certainly have some of these, can you remove the existing ones if you are consolidating.

Yes, these are already moved out of the giant `.test_operators_datetimelike` 

> these also seem oddly specific, would parametrize on Timedelta, TDI as well here

Not sure about TDI, but definitely can parametrize over scalar timedelta variants.